src/annotations/AnnotationManager.php

```diff
@@ -549,3 +577,5 @@ public function getPropertyAnnotations($class, $property = null, $type = null)
         }
     }
 }
+
+AnnotationManager::$traitsSupported = version_compare(PHP_VERSION, '5.4.0', '>=');
```

Comment:
That sure looks like an odd place for storing the code. Why we can't:

* have a regular property instead of static one
* set it in constructor

The is only once `AnnotationManager` class instance per script run so no need to do any static things.